src/garage/tf/baselines/gaussian_cnn_baseline.py

```diff
@@ -199,9 +198,14 @@ def __init__(self,
         self._initialize()
 
     def _initialize(self):
-        input_var = tf.compat.v1.placeholder(tf.float32,
-                                             shape=(None, ) +
-                                             self._input_shape)
+        if isinstance(self._env_spec.observation_space, akro.Image):
+            input_var = tf.compat.v1.placeholder(tf.uint8,
```

Comment:
sorry i take this back as i wasn't thinking very clearly. you should follow what `CategoriaCNNPolicy` does (i.e. use float32): https://github.com/rlworkgroup/garage/blob/master/src/garage/tf/policies/categorical_cnn_policy.py#L123

I think your original version was fine.